swagger.yaml

```diff
@@ -4021,6 +4020,28 @@ definitions:
       type:
         $ref: '#/definitions/image_type'
 
+  host_static_network_config:
```

Comment:
https://github.com/openshift/assisted-service/blob/master/docs/user-guide/restful-api-guide.md#setting-static-network-config needs to be updated to match the new design. pls create a ticket and assign it to me (i'll do that once we're done with pending work around it)